src/SprykerEco/Zed/PunchoutCatalogs/Dependency/Service/PunchoutCatalogsToUtilDateTimeServiceInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs\Dependency\Service;
+
+interface PunchoutCatalogsToUtilDateTimeServiceInterface
+{
+    /**
+     * @param \DateTime|string $date
```

Comment:
Why `\DateTime|string`? Those two types are different & we follow strict types rule.

Reply:
https://spryker.atlassian.net/wiki/spaces/RFC/pages/13434886/ACCEPTED+RFC+Bridges
`Bridge methods have exactly the same signature as their interface.`